### DISCUSSION TITLE
Using @entangle('something').live on arrays triggers an update immediately after page load while

### DISCUSSION THREAD

**Discussion by paladam1** [livewire/livewire#7178] [7. Bugs (archived)]
I use @ entangle('propertyName').live for multi-selects in the AlpineJS x-data property.

If the entangled property is an array and it is set to be a live entangle, only then, an update is triggered immediately after page load, even when this property is not used anywhere inside the component.

`<div x-data="{ something: @entangle('randomProperty').live }">{{-- Nothing here --}}</div>`

In the above example, if the randomProperty is an array, it sends an update, but if randomProperty is not an array (e.g. bool) the update is not sent.

How can I prevent the immediately firing update on arrays as well?

Thanks for your help.

**Comment by aldozumaran** [comment]
i have the same problem

```
Component
class Search extends Component
{
    public array $multi = [];
    ...
}


blade
<select multiple x-data="{
      value: @entangle('multi').live
  }"></select>


```
![image](https://github.com/livewire/livewire/assets/3183704/83d201a6-964c-4811-b126-1f9f848ec55a)

**Reply by sueboy19** [reply]
https://livewire.laravel.com/docs/wire-model#select-dropdowns

```html
<select multiple wire:model.live='multi'>
</select>
```

**Reply by aldozumaran** [reply]
we have the problem with @entangle('multi').live

```
<div x-data="{
      value: @entangle('multi').live
  }">
     <select multiple x-model="value"></select>
</div>

// or just a div

<div x-data="{
      value: @entangle('multi').live
  }">
</div>

```

**Reply by sueboy19** [reply]
```html
<div x-data="{ ttt: @entangle('ttt').live, }">

<input x-model="ttt"> {{ $ttt }}
```
Work OK!

**Reply by aldozumaran** [reply]
It works but the entangle.live triggers an update request immediately after the Page load.

https://user-images.githubusercontent.com/3183704/278855398-83d201a6-964c-4811-b126-1f9f848ec55a.png

**Comment by SaeedKhader** [comment]
I'm experiencing the same issue. Additionally, there's a related problem causing infinite update requests, as detailed in issue #7048.

**Comment by robsontenorio** [comment]
Have you found any solution? It happens on latest 3.2.6

**Comment by gbrits** [comment]
So far I have found removing the .live suffix which makes it the new default defer can rectify some issues - but I am still experiencing a great deal of teething pains going from Livewire 2 to Livewire 3 on my project
